recipes/open62541/all/conanfile.py

```diff
@@ -36,7 +41,7 @@ class Open62541Conan(ConanFile):
         "discovery": [True, False, "With Multicast"],
         "discovery_semaphore": [True, False],
         "query": [True, False],
-        "encryption": [False, "openssl", "mbedtls-apache", "mbedtls-gpl"],
+        "encryption": [False, "openssl", "mbedtls"],
```

Comment:
Yes, good riddance!

Reply:
may it break consumers?

Reply:
Yes if `encryption` was explicitly set downstream to these removed values. Same as https://github.com/conan-io/conan-center-index/pull/6457

Reply:
@uilianries do we need to add some special handling for removed options here to avoid breakage for consumers?

Reply:
Since the default option is persevered it only impacts consumers which rebuild the package locally... CCI consumer should be safe